android_mic_streaming/README.md

```diff
@@ -0,0 +1,17 @@
+# Android Microphone Streaming
+
+Android demo application that streams audio from the microphone to deepspeech and transcribes it.
+
+## Usage
+
+- Download the pre-trained English model and extract it:
+```
+curl -LO https://github.com/mozilla/DeepSpeech/releases/download/v0.6.0/deepspeech-0.6.0-models.tar.gz
```

Comment:
This should be updated to `0.6.1`

Reply:
(TFLite export on v0.6.0 was badly broken, v0.6.1 fixes it).

Reply:
Done